packages/react-composites/tests/browser/call/UFD.test.ts

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Let's name this `UserFacingDiagnostics.test.ts`. The acronym `UFD` might be confusing without context.